research-software-directory/RSD-as-a-service
# 677 cancel image delete

# Fix cancel avatar removal

Closes #677 

Changes proposed in this pull request:
* In the contributor section of edit software, the user can cancel avatar removal in the Edit contributor modal
* In team member section of edit project, the user can cancel avatar removal in the Edit team member modal
* unit test are extended

How to test:
* `make start` to build everything and create test cases
* login as rsd_admin to be able to edit existing software/project, or create new software and project where you are admin of
* navigate to edit software, contributors section. If there are contributors with image, edit the contributor
   * click on remove image button and then use CANCEL instead of SAVE. The avatar should not be removed
   * repeat the process but when using SAVE the avatar image should be removed
   * adding new contributors and removing should also work as expected
* the same fix is applied to team members section of edit project.

PR Checklist:
